calendar-app/src/pages/calendar/MainContent.tsx

```diff
@@ -0,0 +1,53 @@
+import WeekViewTable from "./MainContent/WeekViewTimeframe";
+import CalendarPanel from "./MainContent/CalendarSidePanel";
+import TimeframeToday from "./MainContent/TimeframeToday";
+import RightSidePanel from "./RightSidePanel";
+
+const calendarPanelWidth = "256px";
+const rightSidePanelWidth = "56px";
```

Comment:
very minor, but you can often find primitive value constants named upper snake case `CALENDAR_PANEL_WIDTH = "256px"`